packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -162,6 +196,11 @@ DrawerModalComponent.propTypes = {
   isOpen: PropTypes.bool
 };
 
+DrawerModalComponent.defaultProps = {
+  focusOnMount: true,
+  restoreFocus: true
```

Comment:
Compare to https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/elements/TooltipModal/TooltipModal.tsx#L158-L159